apps/docs/docs/ref/kotlin/installing.mdx

```diff
@@ -17,54 +17,19 @@ custom_edit_url: https://github.com/supabase/supabase/edit/master/web/spec/supab
                 <img src="https://img.shields.io/maven-central/v/io.github.jan-tennert.supabase/supabase-kt?label=experimental" alt="latest supabase-kt version"/>
             </a>
         </div>
-        Add dependency to your build file:
-        <Tabs
-            size="small"
-            type="underlined"
-            defaultActiveId="kotlin"
-            queryGroup="build-file"
-        >
-            <TabPanel id="kotlin" label="build.gradle.kts">
-
-                ```kotlin
-                implementation("io.github.jan-tennert.supabase:[module]:VERSION")
-                ```
 
-            </TabPanel>
-            <TabPanel id="groovy" label="build.gradle">
-
-                ```groovy
-                implementation 'io.github.jan-tennert.supabase:[module]:VERSION'
-                ```
-
-            </TabPanel>
-            <TabPanel id="xml" label="pom.xml">
-
-                ```xml
-                <dependency>
-                    <groupId>io.github.jan-tennert.supabase</groupId>
-                    <artifactId>[module]</artifactId>
-                    <version>VERSION</version>
-                </dependency>
-                ```
-
-            </TabPanel>
-        </Tabs>
+        Add dependency to your build file using the BOM.
 
         The available modules are:
         - [**gotrue-kt**](https://github.com/supabase-community/supabase-kt/tree/master/GoTrue)
         - [**realtime-kt**](https://github.com/supabase-community/supabase-kt/tree/master/Realtime)
         - [**storage-kt**](https://github.com/supabase-community/supabase-kt/tree/master/Storage)
         - [**functions-kt**](https://github.com/supabase-community/supabase-kt/tree/master/Functions)
         - [**postgrest-kt**](https://github.com/supabase-community/supabase-kt/tree/master/Postgrest)
-        - [apollo-graphql](https://github.com/supabase-community/supabase-kt/tree/master/plugins/ApolloGraphQL) - Creates an [Apollo GraphQL Client](https://github.com/apollographql/apollo-kotlin) for interacting with the Supabase API.
-        - [compose-auth](https://github.com/supabase-community/supabase-kt/tree/master/plugins/ComposeAuth) - Provides easy Native Google & Apple Auth for Compose Multiplatform targets.
-        - [compose-auth-ui](https://github.com/supabase-community/supabase-kt/tree/master/plugins/ComposeAuthUI) - Provides UI Components for Compose Multiplatform.
-        - [coil-integration](https://github.com/supabase-community/supabase-kt/tree/master/plugins/CoilIntegration) - Provides a [Coil](https://github.com/coil-kt/coil) Integration for displaying images stored in Supabase Storage.
```

Comment:
If we remove GraphQL here, we should probably also remove the GraphQL snippet in the "Initializing" section

Reply:
It's really up to you, but I personally think it's alright to leave the code snippet in the `initializing` section as it is still a valid example, and we still have the mentions of those additional plugins in the docs just all bundled together.